docs/guides/getting-started-with-opentelemetry.md

```diff
@@ -0,0 +1,377 @@
+---
+weight: 5
+title: How to use opentelemetry metrics with VictoriaMetrics
```

Comment:
opentelemetry -> OpenTelemetry

Reply:
can add information about spanmetrics processor, which allows to convert traces to metrics as a temporary workaround for users who wants use VM for traces as well
https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/connector/spanmetricsconnector/README.md